libs/components/Banner/index.js

```diff
@@ -12,25 +12,12 @@ var _propTypes = _interopRequireDefault(require("prop-types"));
 
 var _classnames = _interopRequireDefault(require("classnames"));
 
+require("./styles.css");
```

Comment:
Looks like here there is a js version and a typescript version side by side? Is that the intention?